src/setup.js

```diff
@@ -21,7 +22,13 @@ export function setupSDK(components: $ReadOnlyArray<SetupComponent<mixed>>) {
   const existingVersion = existingNamespace && existingNamespace.version;
 
   if (existingNamespace) {
-    if (existingNamespace[INTERNAL_DESTROY_KEY]) {
+    if (existingVersion.startsWith("6.")) {
```

Comment:
Is there a scenario where a merchant's code could be using v6 on `window.paypal`, but then this changes it to `window.paypal.v6` and breaks their code?

Reply:
Correct, I think the tradeoff for v6 defaulting to `window.paypal` is that we have to teach merchants using both SDKs to access v6 through `window.paypal.v6`.